lib/rubocop/cop/style/multiline_method_call_brace_layout.rb

```diff
@@ -84,6 +84,15 @@ def children(node)
 
           args
         end
+
+        def ignored_literal?(node)
+          single_line_ignoring_receiver?(node) || super
+        end
+
+        def single_line_ignoring_receiver?(node)
```

Comment:
This is called from `ignored_literal?`, which is called in the included module that I originally edited. That module applies to 4 cops, but moving the code here makes it clear this check is only relevant to this cop, and let's us drop the type check from the earlier commit